csharp/rtl/Constants.cs

```diff
@@ -59,9 +59,9 @@ public struct Constants
         internal static readonly Regex ContentPatternBinary = new Regex(MatchModeBinary,
             RegexOptions.CultureInvariant | RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        public const string DefaultApiVersion = "4.0";
+        public const string DefaultApiVersion = "3.1";
```

Comment:
This seems to have been auto-generated from an old `looker.ini` setting. Need to fix this